### DISCUSSION TITLE
custom-model-binders-and-request-validation/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#44] [Blog comments]
# custom-model-binders-and-request-validation/

https://blogs.taiga.nl/martijn/posts/custom-model-binders-and-request-validation/

**Comment by disqus-comments-migrator** [comment]
<em>Sam Naseri commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 28 2011, 10:40</em>

---

<p>Hi, your solution solved my same problem. Thank you for sharing it with us.</p>

**Comment by disqus-comments-migrator** [comment]
<em>Deepak commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 21 2012, 06:15</em>

---

<p>Could you please elaborate on "it’s possible to add an [ValidateInput(false)] attribute to the controller action that accepts the model with HTML content, but this turns off validation for all model properties". For a model I set [ValidateInput(false)] but still model field validations (like StringLength(3) or RegularExpression etc.) work as intended.</p>

**Reply by CarlosHAraujo** [reply]
This is very old solution, probably the asp.net team already fixed the validation to take per property instead of per model.

**Comment by disqus-comments-migrator** [comment]
<em>martijn commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 21 2012, 06:50</em>

---

<p>With "turns off validation for all model properties" I meant that the default security checks (anti-XSS etc) in ASP.NET are turned off for all model properties, not your own validation attributes.</p>

**Comment by disqus-comments-migrator** [comment]
<em>zenglei commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at mei 17 2012, 09:46</em>

---

<p>Nice</p>

**Comment by disqus-comments-migrator** [comment]
<em>[Carlos Araujo](https://disqus.com/by/disqus_ir9ssJgnav/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at augustus 01 2015, 12:12</em>

---

<p>Your code solves my problem and save me so much precious time.<br>Thank you for sharing, God bless you.</p>
